plugins/plugin-template/static/static_example_file.js

```diff
@@ -0,0 +1,11 @@
+// @Amplication Plugin Template
+// This is an example static file, it should be removed after cloning the template.
+
+/**
+ * This file is a static file that can be added/manipulated and inserted to the Amplication build
+ */
+const exampleFunction = () => {
+  console.log("I'm an example static function");
```

Comment:
if we want to show an example of a static file, we should probably "load" this file from one of the events - currently, the existence of this file does not impact the plugin behavior and it may be more confusing than helpful